.travis.yml

```diff
@@ -103,6 +103,10 @@ script:
   - cd tools && python3 -u build_release_files.py
   - cd ..
 
+  # Check if there's any board missing in TRAVIS_BOARDS
+  - cd tools && python3 -u travis_new_boards_check.py
+  - cd ..
+
```

Comment:
I would move this into the `before_script:` section. Line 73 will work fine, so that we fail fast before downloading any toolchains and building all of the boards.

```
- sudo dpkg --add-architecture i386

# Check if there's any board missing in TRAVIS_BOARDS
- cd tools && python3 -u travis_new_boards_check.py
- cd ..
```

Reply:
Doh! I gave you bad advice. I forgot that the `sh` module is needed, and this placement runs before `pip3 sh`. So anywhere between line 83 and line 93 should do fine. We'll just have to accept downloading gcc.

Reply:
Yes I just noticed it that, I will fix it once I arrive home.